jib-core/src/main/java/com/google/cloud/tools/jib/image/json/HistoryEntry.java

```diff
@@ -105,4 +154,17 @@ public boolean equals(Object other) {
   public int hashCode() {
     return Objects.hash(author, creationTimestamp, createdBy, comment, emptyLayer);
   }
+
+  @Override
+  public String toString() {
```

Comment:
Is this used?

Reply:
Whoops, added that to test something and forgot to get rid of it.